examples/Rules-class-StoppingPatientsNearDose.R

```diff
@@ -1,8 +1,8 @@
 
-# As example, here is the rule for: 
-#   stopping the study if at least 9 patients were dosed at a dose within (1 +/- 0.2) 
-#   of the next best dose
-
-myStopping <- StoppingPatientsNearDose(nPatients = 9,
-                                       percentage = 0.2)
+# As example, here is the rule for stopping the study if at least 9 patients
+# were dosed at a dose within (1 +/- 0.2) of the next best dose.
```

Comment:
is this then 0.2 %? I wonder if this example should be modified to 20?

Reply:
Yeah, thanks Daniel, this is a mistake.